ods_ci/tests/Tests/1100__data_science_pipelines/1100__data-science-pipelines-api.robot

```diff
@@ -22,7 +22,7 @@ ${URL_TEST_PIPELINE_RUN_YAML}=                 https://raw.githubusercontent.com
 *** Test Cases ***
 Verify Pipeline Server Creation With S3 Object Storage
     [Documentation]    Creates a pipeline server using S3 object storage and verifies that all components are running
-    [Tags]    Smoke
+    [Tags]    Smoke    DataSciencePipelines
```

Comment:
This is not needed: we used to have the DataSciencePipelines tag but recently I spitted it to DataSciencePipelines-Backend, DataSciencePipelines-Dashboard and DataSciencePipelines-IDE. They are usually set at the top of the test suites, in the `Test Tags` section

If you want to run all DataSciencePipelines tests, you can do it with:
 `--extra-robot-args '-i DataSciencePipelines-*  '`